build.install4j

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<install4j version="6.1.2" transformSequenceNumber="5">
+<install4j version="6.1.5" transformSequenceNumber="5">
```

Comment:
I tested building the files using your updated build.gradle file so we should be fine...
I built the files using gradlew anyways, since there is no other way unless changing the install4j file